IBAnimatable/AnimatablePresentationController.swift

```diff
@@ -45,9 +50,12 @@ public class AnimatablePresentationController: UIPresentationController {
 
 private extension AnimatablePresentationController {
 
-  func setupDimmingView() {
-    let tap = UITapGestureRecognizer(target: self, action: #selector(chromeViewTapped))
+  func configureDimmingView() {
+    var tap = UITapGestureRecognizer(target: self, action: #selector(dimmingViewTapped))
     dimmingView.addGestureRecognizer(tap)
+    tap = UITapGestureRecognizer(target: self, action: #selector(dimmingViewTapped))
```

Comment:
I thought about again. In the memory model, using one var or two let are same. Because it will be two instances and retained by two different views. I am OK with that now.